### ISSUE TITLE
[CHORE] Replace MobilePlayer by something better (updated)

### ISSUE THREAD

**Issue by rafaelks** [RocketChat/Rocket.Chat.iOS#696]

**Comment by artrmz** [comment]
I wonder what is wrong with Mobile Player? Is it too heavy? Any ideas for replacement?

**Comment by rafaelks** [comment]
@artrmz Last time I checked, was using deprecated methods and old version of Swift. We don't have any ideas right now... we're open for suggestions. :-)

**Comment by artrmz** [comment]
https://github.com/VeinGuo/VGPlayer

This one seems to be updated to Swift 4, maybe can give it a try.

**Comment by cardoso** [comment]
@artrmz

I see one problem: It seems to be lacking a "share" feature, like the one MobilePlayer has.

**Comment by rafaelks** [comment]
@cardoso @artrmz Yes... we could fork some of the libraries to our own org and support it, adding features and bug fixes that we may need.
